docs/source/techspecs/device_disasm_interface.rst

```diff
@@ -195,9 +195,9 @@ the config class to give the information.
 ----------------
 
 There currently is no way for the debugger GUI to add per-core
-configuration.  It is needed for in particular the s2650 and the
-saturn cores.  It should go through the cpu core class itself, since
-it's pulled from the config struct.
+configuration.  It is particularly needed for the s2650 and Saturn
+cores. It should go through the cpu core class itself, since it's
+pulled from the config struct.
```

Comment:
This is a result of French grammar leaking into English ;)

It would be better phrased as “In particular, it is needed for the s2650…”